packages/@ama-sdk/schematics/schematics/helpers/tree-glob.ts

```diff
@@ -15,9 +15,9 @@ const walkThroughDir = (
   parents = ''
 ): string[] => {
   const folders = dir.subdirs.map((d) => `${parents}/${d}`);
-  const matchingFolders = folders.filter(filterFunction);
+  const matchingFolders = folders.filter((value, index, array) => filterFunction(value, index, array));
```

Comment:
is that needed ?

Reply:
yes because `<array>.filter` can take up to 4 arguments